data-plane/config/broker/500-dispatcher.yaml

```diff
@@ -31,6 +33,7 @@ spec:
       name: kafka-broker-dispatcher
       labels:
         app: kafka-broker-dispatcher
+        app.kubernetes.io/component: kafka-dispatcher
```

Comment:
should we keep the `-broker-` ? I know it's same image, but it is reused on different components